includes/class-wc-siftscience-comm.php

```diff
@@ -50,15 +51,15 @@ public function post_event( $data ) {
 		public function post_label( $user_id, $is_bad ) {
 			$data = array(
 				'$api_key'    => $this->options->get_api_key(),
-				'$is_bad'     => ( $is_bad ? 'true' : 'false' ),
+				'$is_bad'     => ( $is_bad ? true : false ),
 				'$abuse_type' => 'payment_abuse',
 			);
 
 			$url  = str_replace( '{user}', rawurlencode( $user_id ), self::LABELS_URL );
 			$args = array(
 				'headers' => self::HEADERS,
 				'method'  => 'POST',
-				'body'    => $data
+				'body'    => $data,
 			);
 
```

Comment:
testing GitHub website just ignore this comment

Reply:
this is inconvenient

Reply:
I see this comment.